backend-api/src/functions/asyncActiveSession/asyncActiveSessionHandler.ts

```diff
@@ -55,20 +56,37 @@ export async function lambdaHandlerConstructor(
     logger.log("INTERNAL_SERVER_ERROR", {
       errorMessage: getSubFromTokenResult.value.errorMessage,
     });
-    return serverError500Response;
+    return serverErrorResponse;
+  }
+
+  const sessionService = dependencies.sessionService(config.SESSION_TABLE_NAME);
+  const getActiveSessionResult =
+    await sessionService.getActiveSession("mockSub1"); // temporary sub until we have implemented the logic to extract the sub from the JWT
+
+  if (getActiveSessionResult.isError) {
+    logger.log("INTERNAL_SERVER_ERROR", {
+      errorMessage: getActiveSessionResult.value.errorMessage,
+    });
+    return serverErrorResponse;
+  }
+
+  const session = getActiveSessionResult.value;
+  if (session === null) {
+    return notFoundResponse;
   }
 
   logger.log("COMPLETED");
 
   return {
+    headers: { "Content-Type": "application/json" },
     statusCode: 200,
-    body: "Hello, World",
+    body: JSON.stringify(session),
```

Comment:
Please may we explicitly have this as a full object? Reasons for asking (given this functionality is already good):
- I'd like to be able to see when looking at the handler code very clearly what response we're giving
- it makes me nervous that we're dynamically tying this response to the behaviour of a method in a different service? I know our type system/mocks would help us catch this, but I'd rather be declarative if poss

Reply:
I don't agree with that. This would mean creating exactly the same object we are creating in `SessionService` and then doing exactly the same checks (is redirect_uri defined or not) in the handler. We will basically write the same code twice. We need to be able to trust our code and trust that people aren't making silly changes that will break everything. And our unit tests and API tests make sure that this doesn't happen.

Reply:
If we are going to do that then we might as well just move the entire logic to the handler

Reply:
fine to resolve this - thanks for the viewpoint